### PR TITLE
Fix WW prices

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: objects/rct2ww/scenery_large/rct2.ww.eiffel.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
     "sourceGame": "rct2ww",
     "objectType": "scenery_large",
     "properties": {
-        "price": 190,
-        "removalPrice": -150,
-        "cursor": "CURSOR_STATUE_DOWN",
+        "price": 204,
+        "removalPrice": -168,
+        "cursor": "CURSOR_HOUSE_DOWN",
```
Comment (AaronVanGeffen) [inline objects/rct2ww/scenery_large/rct2.ww.eiffel.json:11]: Not sure how the house cursor is more appropriate for the Eiffel Tower. Is there a rationale?

File: objects/rct2ww/scenery_large/rct2.ww.gwoctur1.json [hunk 2]
```diff
@@ -92,7 +92,6 @@
             "pl-PL": "Wielki Mur Chiński - wieżyczka prosta",
             "ru-RU": "Фрагмент Великой Стены",
             "pt-BR": "Reta de torre da Grande Muralha da China",
-            "ja-JP": "万里の長城 Turret Straight"
```
Comment (Gymnasiast) [inline objects/rct2ww/scenery_large/rct2.ww.gwoctur1.json:95]: revert

File: objects/rct2ww/scenery_large/rct2.ww.gwoctur2.json [hunk 3]
```diff
@@ -92,7 +92,6 @@
             "pl-PL": "Wielki Mur Chiński - wieżyczka narożna",
             "ru-RU": "Фрагмент Великой Стены",
             "pt-BR": "Canto de Torre da Grande Muralha da China",
-            "ja-JP": "万里の長城 Turret Corner"
```
Comment (Gymnasiast) [inline objects/rct2ww/scenery_large/rct2.ww.gwoctur2.json:95]: revert

File: objects/rct2ww/scenery_large/rct2.ww.mbskyr02.json [hunk 4]
```diff
@@ -82,7 +82,7 @@
             "ko-KR": "마천루 끝",
             "pl-PL": "Koniuszek drapacza chmur",
             "ru-RU": "Верхушка",
-            "pt-BR": "Topo de Arranha-céu"
+            "pt-BR": "Topo de Arranha-céu",
```
Comment (Gymnasiast) [inline objects/rct2ww/scenery_large/rct2.ww.mbskyr02.json:85]:
```suggestion
            "pt-BR": "Topo de Arranha-céu"
```

File: objects/rct2ww/scenery_large/rct2.ww.pagodatw.json [hunk 5]
```diff
@@ -92,7 +92,6 @@
             "pl-PL": "Dach pagody",
             "ru-RU": "Пагода",
             "pt-BR": "Torre Pagode",
-            "ja-JP": "パゴダ"
```
Comment (AaronVanGeffen) [inline objects/rct2ww/scenery_large/rct2.ww.pagodatw.json:95]: No need to remove this.

Comment (Gymnasiast) [inline objects/rct2ww/scenery_large/rct2.ww.pagodatw.json:95]: revert

Comment (Gymnasiast) [reply objects/rct2ww/scenery_large/rct2.ww.pagodatw.json]: This PR should not touch Japanese at all, but some changes slipped through my initial filtered patch. Fixed now.

File: objects/rct2ww/scenery_large/rct2.ww.eiffel.json (hunk 1 above)
Comment (Gymnasiast) [reply objects/rct2ww/scenery_large/rct2.ww.eiffel.json:11]: The house cursor appears to be used for any large building. The Colosseum also uses it, for example.

**Pull request merged**